examples/dmd/wave_equation.cpp

```diff
@@ -0,0 +1,685 @@
+ /*
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+//
+//                       libROM MFEM Example: Wave_Equation (adapted from ex23.cpp)
+//
+// Compile with: make wave_equation
+//
+// Sample runs:  wave_equation
+//               wave_equation -o 4 -tf 5
+//               wave_equation -m ../data/square-disc.mesh -o 2 -tf 2 --neumann
+//               wave_equation -m ../data/disc-nurbs.mesh -r 3 -o 4 -tf 2
+//               wave_equation -m ../data/inline-hex.mesh -o 1 -tf 2 --neumann
+//               wave_equation -m ../data/inline-tet.mesh -o 1 -tf 2 --neumann
+//
+// =================================================================================
+//
+// Sample runs and results for DMD:
+//
+// Command 1:
+//   wave_equation -o 4 -tf 5
```

Comment:
add -nwinsamp option here. By the way, what happens when -nwinsamp is not provided as a command line option?

Reply:
When `-nwinsamp` is not provided as a command line argument, it defaults to `std::numeric_limits<int>::max()`. This is in effect 1 window for the time-windowing DMD.

Reply:
Done.